vendor/gopt/gopt.c

```diff
@@ -80,7 +80,7 @@ void add_argument_value(struct option options[], int option_index, char* value)
   int arg_index = options[option_index].count - 1;
 
   if (options[option_index].flags & GOPT_REPEATABLE_VALUE) {
-    if (arg_index < options[option_index].max_args) {
+    if (arg_index < (int)options[option_index].max_args) {
```

Comment:
You might consider checking if there is a newer "upstream" gopt that already fixed this. If so, we could just bump the library up.